cloudformation/database/serverless-database.yaml

```diff
@@ -0,0 +1,345 @@
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      -
+        Label:
+          default: "Database Cluster Configuration"
+        Parameters:
+          - ClusterAdmin
+          - AdminPasswordSecret
+          - MinCapacity
+          - MaxCapacity
+      -
+        Label:
+          default: "Network Configuration"
+        Parameters:
+          - Vpc
+          - PrivateSubnetOne
+          - PrivateSubnetTwo
+    ParameterLabels:
+      Vpc:
+        default: "Which VPC should this be deployed to?"
+Parameters:
+  AdminPasswordSecret:
+    Description: Name or ARN of SecretManager secret for Admin Password.
+    Type: String
+    MinLength: 2
+    MaxLength: 1024
+  ClusterAdmin:
+    Description: Database Cluster Admin User Name.
+    Type: String
+    Default: clusteradmin
+    MinLength: 3
+    MaxLength: 64
+  Vpc:
+    Description: VPC ID to create the Database Cluster in (leave BLANK to create a new one).
+    Type: String
+    # Type: AWS::EC2::VPC::Id
+    Default: ""
+  PrivateSubnetOne:
```

Comment:
Is it necessary for the subnets to be private?
I mean: from a security standpoint that's better because we do not want to expose a public ip for the database, but that is not mandatory.
What do you think?

Reply:
Is there a good reason to not make them private? My thought is that by default, they should be as locked down as possible unless there is a reason not to?

Reply:
Ah OK, I think I understand what you are saying now. I've renamed the parameters.